pmxbot/stack.py

```diff
@@ -200,14 +203,16 @@ def save_items(self, topic, items):
     "shuffle": "!stack shuffle <topic[index]>: Shuffle items from the given "
                "topic into the the given (1-based) index order "
                "(default: random)",
-    "topics": "!stack topics <[index]>: Show topic names, numbered in "
+    "topics": "!stack topics|list <[index]>: Show topic names, numbered in "
```

Comment:
hmm. Let's settle on a name. Either topics or list should suffice; no need to be ambiguous about it.